koku/api/currency/view.py

```diff
@@ -46,7 +38,7 @@ def get_currency(request):
 
     """
     try:
-        data = load_currencies(CURRENCY_FILE_NAME)
+        data = load_currencies_from_file()
```

Comment:
@dccurtis we should also add to the response the selected currency from settings.

@ddonahue007 @dlabrecq - is that right?